src/sourmash/commands.py

```diff
@@ -169,16 +169,24 @@ def compare(args):
         similarity = compare_all_pairs(siglist, args.ignore_abundance,
                                        n_jobs=args.processes, return_ani=return_ani)
 
+    # if distance matrix desired, switch to 1-similarity
+    if args.distance_matrix:
+        similarity = 1 - similarity
```

Comment:
well. that's fine. but it is deeply confusing that the distance matrix will be named similarity. I get why it is tho so i accept this chaos and will move on with my life.

Reply:
nah, you were right. renamed to `matrix` in 6091d39a.